app/javascript/components/pages/Songs.js

```diff
@@ -1,38 +1,49 @@
 import React, { Component } from "react";
-import { thead, tr, th, tbody, Table, hover } from "reactstrap";
+import { thead, tr, th, tbody, Table } from "reactstrap";
+
 class Songs extends Component {
   render() {
     const { songs } = this.props;
     return (
       <>
-        <h1>This is songs</h1>
+        <th>
+          <div className="barcode">
+            <h1>Library</h1>
+          </div>
 
-        <Table hover>
-          <thead>
-            <tr>
-              <th>Artist</th>
-              <th>Song</th>
-              <th>Genre</th>
-            </tr>
-          </thead>
-          {songs &&
-            songs.map((song, i) => {
-              return (
-                <tbody key={i}>
-                  <tr>
-                    {/* <th scope="row"></th> */}
-                    <td>{song.song_artist}</td>
-                    <td>{song.song_name}</td>
-                    <td>{song.genre}</td>
-                  </tr>
-                </tbody>
-              );
-            })}
-        </Table>
+          <Table hover>
+            <thead>
+              <tr>
+                <th>COVER</th>
+                <th>ARTIST</th>
+                <th>SONG</th>
+                <th>GENRE</th>
+              </tr>
+            </thead>
+            <br></br>
 
-        {/* <ul>
-                   <li key={song.id}> {song.song_name} , {song.song_artist} , {song.genre}</li>
-                 </ul> */}
+            {songs &&
+              songs.map((song, i) => {
```

Comment:
A fancy way of doing this is `{songs?.map....}` javascript introduced "optional chaining" a couple versions ago and it's super handy. just an FYI 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining